llvm/test/SYCLLowerIR/PropagateAspectUsagePass/empty.ll

```diff
@@ -0,0 +1,20 @@
+; RUN: opt --PropagateAspectUsage < %s -S | FileCheck %s
```

Comment:
I think that we should also add a test where a function uses two types that share the same aspect attached to them - to check that we don't emit duplicates in `!intel_used_aspects` metadata

Reply:
added one.